[2021-07-21 21:25:08,090] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: scrape_nasdaq_ftp.nasdaq_symbols_download_files 2021-07-21T19:25:06.686781+00:00 [queued]>
[2021-07-21 21:25:08,093] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: scrape_nasdaq_ftp.nasdaq_symbols_download_files 2021-07-21T19:25:06.686781+00:00 [queued]>
[2021-07-21 21:25:08,094] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-07-21 21:25:08,094] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-07-21 21:25:08,094] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-07-21 21:25:08,105] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): nasdaq_symbols_download_files> on 2021-07-21T19:25:06.686781+00:00
[2021-07-21 21:25:08,107] {standard_task_runner.py:52} INFO - Started process 20760 to run task
[2021-07-21 21:25:08,111] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'scrape_nasdaq_ftp', 'nasdaq_symbols_download_files', '2021-07-21T19:25:06.686781+00:00', '--job-id', '65', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/nasdaq_dag.py', '--cfg-path', '/tmp/tmp7rq5crsh', '--error-file', '/tmp/tmpzt35xrbq']
[2021-07-21 21:25:08,111] {standard_task_runner.py:77} INFO - Job 65: Subtask nasdaq_symbols_download_files
[2021-07-21 21:25:08,137] {logging_mixin.py:104} INFO - Running <TaskInstance: scrape_nasdaq_ftp.nasdaq_symbols_download_files 2021-07-21T19:25:06.686781+00:00 [running]> on host DESKTOP-4B4VMOV.localdomain
[2021-07-21 21:25:08,169] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=scrape_nasdaq_ftp
AIRFLOW_CTX_TASK_ID=nasdaq_symbols_download_files
AIRFLOW_CTX_EXECUTION_DATE=2021-07-21T19:25:06.686781+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-21T19:25:06.686781+00:00
[2021-07-21 21:25:08,169] {taskinstance.py:1501} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/yaysh/anaconda3/envs/airflow/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1157, in _run_raw_task
    self._prepare_and_execute_task_with_callbacks(context, task)
  File "/home/yaysh/anaconda3/envs/airflow/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1331, in _prepare_and_execute_task_with_callbacks
    result = self._execute_task(context, task_copy)
  File "/home/yaysh/anaconda3/envs/airflow/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1361, in _execute_task
    result = task_copy.execute(context=context)
  File "/home/yaysh/anaconda3/envs/airflow/lib/python3.8/site-packages/airflow/operators/python.py", line 150, in execute
    return_value = self.execute_callable()
  File "/home/yaysh/anaconda3/envs/airflow/lib/python3.8/site-packages/airflow/operators/python.py", line 161, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/yaysh/airflow/dags/nasdaq_dag.py", line 37, in download_nasdaq_symbols
    print(my_env)
NameError: name 'my_env' is not defined
[2021-07-21 21:25:08,171] {taskinstance.py:1544} INFO - Marking task as FAILED. dag_id=scrape_nasdaq_ftp, task_id=nasdaq_symbols_download_files, execution_date=20210721T192506, start_date=20210721T192508, end_date=20210721T192508
[2021-07-21 21:25:08,201] {local_task_job.py:149} INFO - Task exited with return code 1
